pyproject.toml

```diff
@@ -37,6 +36,7 @@ toml = "^0.10.2"
 colorama = "^0.4.4"
 pyfiglet = "^0.8.post1"
 curlify2 = "^1.0.0"
+kubernetes-asyncio = {git = "https://github.com/opsani/kubernetes_asyncio", rev = "v12.1.2-custom-resource-patch-fix"}
```

Comment:
The patch I'm using in the fork is not upstreamable as it was applied to the generated code rather than updating the code generation models. The upstream library maintainer has [PRed a fix](https://github.com/kubernetes-client/python/pull/959) for this [issue](https://github.com/tomplus/kubernetes_asyncio/issues/68) into the standard kubernetes python client but its unclear when that will be landed and the fork is mainly a temporary workaround.

The library does not release very frequently so keeping up with patching the fork should not be too burdensome and I've also thrown together some automation on the fork to let us know when we need to rebase it: https://github.com/opsani/kubernetes_asyncio/blob/master/.github/workflows/check-upstream-release.yaml

See [ENG-148](https://linear.app/opsani/issue/ENG-148/kubernetes-connector-is-unable-to-patch-custom-resource-objects-due-to) for discussion pertaining to this workaround